tests/src/StringTranslation/StringTranslationTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\service_container\StringTranslation\StringTranslationTest
+ */
+
+namespace Drupal\Tests\service_container\StringTranslation;
+
+use Drupal\service_container\StringTranslation\StringTranslation;
+
+/**
+ * @coversDefaultClass \Drupal\service_container\StringTranslation\StringTranslationTest
```

Comment:
Should be:

\Drupal\service_container\StringTranslation\StringTranslation

Then tests pass.